test-commons/src/test/java/com/sonarsource/checks/verifier/SingleFileVerifierTest.java

```diff
@@ -0,0 +1,56 @@
+/*
+ * SonarQube Analyzer Test Commons
+ * Copyright (C) 2009-2017 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package com.sonarsource.checks.verifier;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import org.junit.Test;
+
+import static java.nio.charset.StandardCharsets.UTF_8;
+
+public class SingleFileVerifierTest {
```

Comment:
Do we assert anything in this test?

Reply:
Yes, it's non regression test to check the interface SingleFileVerifier continue to works.